### ISSUE TITLE
Error running stylelint Error: Column start (11) greater than line length (0) for line 8

### ISSUE THREAD

**Issue by aubreyz** [stylelint/stylelint#4870]
Odd crash: Error running stylelint Error: Column start (11) greater than line length (0) for line 8

Produced by this semi-minimal code

The crash can be prevented by apparently unrelated changes to the code
a) Removing just the bracketed word (don't) in the comment
b) Reducing the first double line space to a single line space (but not the second double line space)
c) Deleting the whole 4 line block at the end starting ul

The line numbers reported do not seem to relate to anything and with longer code do not point to anywhere apparently related

On version 13.6.1, Windows 10

Probably related to https://github.com/AtomLinter/linter-stylelint/issues/176 old report in the atom package related to this linter, and also possibly https://github.com/stylelint/stylelint/pull/1168 (which was reported fixed)

File name styles.less (a related report suggests a similar bug might occur with the same code in less but not css)
```
atom-text-editor.editor {
  .line-number.foldable .icon-right {
    visibility: visible;
  }


}
// NOTE: atom-text-editor, color various stuff (don't)

atom-text-editor {
  .invisible-character {
    color: yellow;
  }

  .syntax--punctuation.syntax--definition.syntax--comment {
    color: #008C3F;
  }
}


ul.tab-bar > li.tab[is='tabs-tab'].active,
ul.tab-bar > li.tab[is='tabs-tab'].active > div.title {
  color: white;
}

```

**Comment by aubreyz** [comment]
On further exploration just deleting the ' in the commented word don't prevents the crash (but that is clearly not the only reason as other code changes which retain the ' do not crash)

**Comment by hudochenkov** [comment]
Please, use issue template because many important things are missing. Also include file name (file extension is important).

Our demo [doesn't have any issues](https://stylelint.io/demo#N4Igxg9gJgpiBcICGAXCBbAtCmAPFmMUAlmgE4B0RpEZABMADoB2ddFANsczJswK7oARjEoAzCByhIhHGO2KRmmMsQDmACxQMWbNgDdiAZ2JDiXFAE94dQydkwA3LroBfFh+bvmAeh90AOQB5ABUAURtUDGw8AmpyABo6SA5aWyRVCH4jOiMUfjExOgAKKAhmAHIUAEpPKKwcfEISch1WBWY7UzlMMA0MpDAceiZ2thTaG0sYDlSAd2d27xcKI0tmFCRcTEwAB35mIf5UYnLV9c3tzFgxblJT5nONrZ3IdHQYDba9ZMlJugAxAAGIEADgAwgBmABiizY3mWzBY-A4FE2QkwQgydAAfHQuGiZABtYwAXiqMiM2BkFQAuhRBihiPoYAlkaj0ZjsXiCeiSUZyeiqei6Qyhsz5HiSPo0aQ5N9fqkyDY5hpSE4WK4QAkQGtnrgELqwEYjNrwOVbmpDaM2IwQGQUTAjHabDa9HbUCpHXwIJgDgBrZgQOZIhB0FAO1kuW0gWQQMD+n2EdC7Kwu8ORtljOh2iZkJPcfRILhQTAaPDpiP8KPZ3MYD4bJMwFNpsNVms-O2wMAcAZM8qY1IJpNQfi7LhgVC8XZkCC7URMp3ponRvTtrM-Nhuzc5kDqINkGDL1c7uvMIwwMD8Jks65jidTqlzUgaa7EQqiT4EIscavOkAnnotKAd4Py0hu7ogN2vZkCcA5xsOQaYEYGi0Cg-TMKWM5zguliYBALJkKosD-jY66rnaEiNmISDoOYeFIaO46KFOfC0UubaZhRIBUQQNF0RwDG+nRJrcGomBqJ8oiKJg-H0Zg-owJYcy0FAlZcbWPEHOKA6ThwYAFp0xbEGpnHVhBMZiNp-bKHpBlIQcRi7IMRD4fOsHkOp5ncVZhw2ZgXA8BkEmwSQX5JkG56bJhGSliQh46aGZEaZ2ICKZYYiwR81yXjBcHKEhxApmhSAbF5HaQR8JBILJMCoPwh5sdlDnMIGwZJRm3maYxTpgJ8JDMOJTmXu+iikJY5UWbujH3ixOCYJ6RW7GhXpyKRnUVTGM3MZO80XnIQy0Ot5Hdb6zapnhRhZGQfWTdxSGxLByHNoofznndp2YIWxlxVkDjIb2KG9PWX7HSlkHYe5VhJgGQYhh9qX7Ze5B7Be-BlL0gNUi1bXw2Zm27kjh35rsaMYzMzbhTjcMdSdiMU8T2CWPOMOtTTCOQXkqiDRFMBzIFR741NdoHKQrO47T4Mxp6Do9Od0MC5iMASIex7ZtLHBzEglj-lNW6ATGeB9amas7qlCH+mtVJIGIwzIexTW8BbVt2nrm6UcQZB5HwTo4KZBt0OBAd2vutCCzYK7q+7yC26IwPvF+doB8BUegUBwvIAQsu8MwDuThe6Z2vMoiu9xMvern2VOS5C2x2QhfIJr2tUiYg09ALpeaeXPQXnREwFXzis28MDfFlrOud+bQ6Jj2ECt+JQiwX1yYXQF3BO8rYcNzwhGT5BFuY3PYmYkvOeD+vtcj2GHpNxP2rcQfs-zyfNc8PzF8iCr4e7mPzeYOgKImRrx4HvGMj9UjP0XjXauy9P5b2vo3ceLcxLt3XqA3cB8cLMGPlA5eb8h511HrfKkACOBAI7vfTSmD5zYJ5rg3gMDeDDxLgg3+OtkIoN4BQjO1DPg4NPshZysDN6q1YcQ9BdYlRljwL0JABcEHF3rpQ1KeZpHbDkINdCDcUJoQkeaBOjZ5Z4UVnA0REdAI3yQeg-WUdDa4GNigU2ZtLKe29jwPIRAk62MDm7GMoczF0Ejs4mMeRLByECgQN46BSpQH-MnECq4g6aSieFVU6pGFfXPCZb+ljm56KvHkaIkNcIrwVh-ER38gnRzYbrQC25o5GxgCbMMVTnEejrr0bIaAsDFLIK2XxkFbhewIO4v2XjnFJLaXuNQB5KkB2lh0lJZVlHBJDlk2AHC25cI-tPcZZsU47jTmwSZkFoJ9geFyHmGTmFKJsHaHeLCM5nI8hcrEVyhEby-kQqxKyYzPPyoOeMiZe6vVSAPd+PBL4sLuYgv+pDyFoN+buf5-kD4gv7oImuNzvl-3nqgkBSKuy5XOfBaez0+5vUxcIr5CCHlKKecSl5pKgWbLUPi3g0TtgooeOtAAjAynsJLlAHwjEgcwx90VvRxXfAVeV-IYvwRfbFYikH-0AcQYBgtZVCuBmCqlTDCEqtxZwzVejuW6UpRk0xOSQB0rNYygFRjNVKxpeYqOuSZV1ODiARpzS3XBJ-h081oYBkuOGT7Dx-tvEnNPNM2ZTipk3Pjg2RxSLY3cBMBsvF2zIUWz2TuA5m4jk+J8tZC5USaqKshcqmFNS1VkI1dw0tfly31hqtcw1tbiGsvZXo3yiVk3to+S6+BMK7WEq0i2gcnLSnGPXutIEGd+3+UrrwfONrFF9rLQOZyh4NjlgvNjc+kKM3ZOlSQ9VpqJ3Loubur8B6nT6syZmm1dKe05q1c2gdaScAdqvl2n5GduCwGeDZdMAAmDOGi1DoUwAAL1EL6FqpAOYxhnU6gWfKM5VWIDVMQdV8iNQxX+6FP9xETpw3hgjDU12WuHda7eMBd4UeoFR+qjVV2yPkTCzdLHqq1XY9ODI96nSPoyae2AjHmPYdYwJwjvBYKDV4DhDytAn01rI4B7ilG5M0ZUKVSSblRBRHzFaip569E6YAI7VjIHO72FafYQoNf+zTcLL1Ns0tZ2z9nIltvU52tz7Ds1Xpk-xmzohfODqfQx2lTHHn3TOpITAP6nQfNQ9NYSxhn79XwmIZC108HHu-nTSCAhhBxzkEgAa4kEOzgsxO8rIh8xIVFeKnmdW54ZbtL06G66G68YztnWdzrYv+uqd2+FjbEVu3qalX1qbxtTKGW432niALRoGSHGZo7AnzMDXbJZqb4mp0SRnImKNUARlMNeJ2S9FIoBbsOiTr74v0u4hdtTV3VBCFu0Z1Tpnh0afuW9vRn38zfZu-NFTJmYvmbi9Jj7DMUZvDMLnFGJHblBdqZpcH8c0ew7M667HYPkdqZIn1GKjZUfcFh0hSK+ruu6jJ-mLgDn-NVpc6Rj1OP6YHRRmzvz7wh01zG9NUHE68foZbHO9x6ZF1I-52p0mMB0a+kFSaLjG7gwJdxyz1GquMYa6pHe-domns1xe1J3XfPkbK7JmdOQKatcDZ1+9vXSuSYO8IE78KCqICbDAwgso-AHCk890zFm-WFFu70aLSJcjtdzBt5BH81YFJKRUmQUs0eeOx4nWn7Z7PhdOYIa5nn9aEUEozoXtexfokBfL7C4LJrPOpVr4L6LRPdsg8R5pDvxgCDS9Xph+XLgEQgFcEAA) with example code, so it might be configuration, or Atom plugin issue.

**Comment by aubreyz** [comment]
Many apologies, I assumed the atom linter simply passed a saved file to stylelint and passed back errors. Clearly not the case. Thank you for pointing to the online demo of which I was not aware. The filetype is less, and I agree does not throw up errors with less and this code. I'm closing this and will post on the https://github.com/AtomLinter/linter-stylelint repo
